src/renderers/scene_renderer.py

```diff
@@ -16,6 +18,9 @@ def render_scene(self):
         scene = bpy.context.scene
         scene.camera = bpy.context.object
 
+        cameracon = CameraController()
+        cameracon.setup_camera("camera_one", (10,0,0), (1.57057,0.00174533,1.57057), "Collection")
```

Comment:
"Collection" should not be a hard-coded string. Instead it should be the `collection` variable which is already being passed during initialisation